internal/app/machined/pkg/controllers/k8s/templates.go

```diff
@@ -473,10 +473,6 @@ kind: ClusterRole
 metadata:
   name: flannel
 rules:
-  - apiGroups: ['extensions']
-    resources: ['podsecuritypolicies']
-    verbs: ['use']
-    resourceNames: ['psp.flannel.unprivileged']
```

Comment:
this was never actually used, as `psp.flannel.unprivileged` policy was never created